Sorting/Sans Pointers/Bubble Sort.cpp

```diff
@@ -0,0 +1,50 @@
+#include <iostream>
+using namespace std;
+//void BubbleSort(int A[]);
+//int swap (int,int);
+
+int A[50],i,j,n,a,b,t;
+
+void BubbleSort(int A[ ], int n)
+{
+//	int i,j,n;
+	for (i=0;i<n;i++)
+	{
+		for (j=0;j<n-i-1;j++)
+		{
+			if (A[j]>A[j+1])
+			{
+				swap (A[j],A[j+1]);
+			}
+		}
+	}
+}
+
+int swap(int a, int b)
```

Comment:
Check your swap function. Finish up the code I asked you to make yesterday night  , and you will yourself realize the mistake made here .